File: pset_1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height; //user input

    // Input 1-8 of height required from user
    do
    {
        height = get_int("Size of pyramid: ");
    }
    while (height > 8 || height < 1);

    // Outer for loop, looping through and creating rows based on height
    for (int row = 0; row < height; row++)
    {
        // Print white space based on number of times outside for loop has run
        for (int space = row + 1; space < height; space++)
        {
            printf(" ");
        }
        // Printing hashes depending on height and number of times outside for loop has run
        for (int hash = height + row + 1; hash > height; hash--)
        {
            printf("#");
        }
        printf("  ");
        for (int hash = height + row + 1; hash > height; hash--)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: pset_6/sentimental-mario-less/mario.py
from cs50 import get_int

# input of 1-8 required from user
while True:
    height = get_int("Height: ")
    if height >= 1 and height <= 8:
        break


# looping through and creating rows based on height
for row in range(height):
    # print whatever is needed at that brick
    for brick in range(row + 1, height + row + 1):
        if brick < height:
            print(" ", end="")
        else:
            print("#", end="")
    # print new line for new row
    print()

